nvidia_triton/datadog_checks/nvidia_triton/check.py

```diff
@@ -0,0 +1,111 @@
+        super(NvidiaTritonCheck, self).__init__(name, init_config, instances)
+        if 'openmetrics_endpoint' not in self.instance:
+            raise ConfigurationError(
+                "The 'openmetrics_endpoint' option is missing in the Nvidia Triton integration configuration."
+            )
+        self.openmetrics_endpoint = self.instance["openmetrics_endpoint"]
+        self.tags = self.instance.get('tags', [])
+
+        # Get the API server port if specified, otherwise use the default 8000.
+        self.server_port = str(self.instance.get('server_port', 8000))
+
+        self.collect_server_info = self.instance.get('collect_server_info', True)
+
+        # Get the base url from the openmetrics endpoint and construct the server info API endpoint.
+        if self.collect_server_info is True:
+            self.base_url = None
+            try:
+                parts = urlparse(self.openmetrics_endpoint)
+                # Delete the /metrics from the url
+                self.base_url = parts._replace(path="")
+                # Replace the openmetrics port by the server port
+                self.server_info_api = self.base_url._replace(netloc=parts.hostname + ':' + self.server_port).geturl()
+
+            except Exception as e:
+                self.log.debug("Unable to determine the base url for server info collection: %s", str(e))
+
+    def get_default_config(self):
+        return {
+            "metrics": [METRICS_MAP],
+            # Rename labels that are reserved in datadog.
+            "rename_labels": {
+                "version": "triton_model_version",
+            },
+        }
+
+    @AgentCheck.metadata_entrypoint
+    def _submit_version_metadata(self):
+        if self.collect_server_info is False:
+            self.log.warning("Collecting server info through API is disabled.")
+            return
+
+        endpoint = urljoin(self.server_info_api, DEFAULT_METADATA_ENDPOINT)
+        response = self.http.get(endpoint)
+
+        if response.ok:
+            data = response.json()
+            version = data.get("version", "")
+            version_split = version.split(".")
+            if len(version_split) >= 3:
+                major = version_split[0]
+                minor = version_split[1]
+                patch = version_split[2]
+
+                version_raw = f'{major}.{minor}.{patch}'
+
+                version_parts = {
+                    'major': major,
+                    'minor': minor,
+                    'patch': patch,
+                }
+                self.set_metadata('version', version_raw, scheme='semver', part_map=version_parts)
+            else:
+                self.log.debug("Malformed Triton Server version format: %s", version)
+        else:
+            self.log.debug("Could not retrieve version metadata.")
+
+    def _check_server_health(self):
+        if self.collect_server_info is False:
+            self.log.warning("Collecting server info through API is disabled.")
```

Comment:
```suggestion
            self.log.debug("Collecting server info through API is disabled.")
```
If the user explicitly disabled collecting server info, we don't need to spam their logs by telling them what they already know. It could be useful for debugging though.